diff -Nru gkrellm-1.0.6/src/gkrellm.h gkrellm-1.0.6.new/src/gkrellm.h
--- gkrellm-1.0.6/src/gkrellm.h	Tue Jan 30 07:16:17 2001
+++ gkrellm-1.0.6.new/src/gkrellm.h	Tue Jan 30 07:18:29 2001
@@ -121,12 +121,12 @@
 
 #define	LOCAL_THEMES_DIR		"/usr/local/share/gkrellm/themes"
 #if !defined(SYSTEM_THEMES_DIR)
-#define	SYSTEM_THEMES_DIR		"/usr/share/gkrellm/themes"
+#define	SYSTEM_THEMES_DIR		"/usr/X11R6/share/gkrellm/"
 #endif
 
 #define	LOCAL_PLUGINS_DIR		"/usr/local/lib/gkrellm/plugins"
 #if !defined(SYSTEM_PLUGINS_DIR)
-#define	SYSTEM_PLUGINS_DIR		"/usr/lib/gkrellm/plugins"
+#define	SYSTEM_PLUGINS_DIR		"/usr/X11R6/lib/gkrellm"
 #endif
 #define	OLD_LOCAL_PLUGINS_DIR	"/usr/local/share/gkrellm/plugins"
 #define	OLD_SYSTEM_PLUGINS_DIR	"/usr/share/gkrellm/plugins"
