diff -ru gkrellm-2.3.0/src/configure gkrellm-2.3.0.my/src/configure
--- gkrellm-2.3.0/src/configure	2007-05-07 01:45:55.000000000 +0200
+++ gkrellm-2.3.0.my/src/configure	2007-09-19 19:05:12.000000000 +0200
@@ -41,6 +41,7 @@
 
 cat << EOF > test.c
 #include <gnutls/openssl.h>
+#include <gcrypt.h>
 #include <errno.h>
 #include <pthread.h>
 
diff -ru gkrellm-2.3.0/src/mail.c gkrellm-2.3.0.my/src/mail.c
--- gkrellm-2.3.0/src/mail.c	2007-09-19 19:08:34.000000000 +0200
+++ gkrellm-2.3.0.my/src/mail.c	2007-09-19 19:06:25.000000000 +0200
@@ -34,6 +34,7 @@
 
 #if defined(HAVE_GNUTLS)
 #include <gnutls/openssl.h>
+#include <gcrypt.h>
 #include <pthread.h>
 #define MD5Init		MD5_Init
 #define MD5Update	MD5_Update
