diff -u -r gkrellm-2.3.0.orig/server/configure gkrellm-2.3.0/server/configure
--- gkrellm-2.3.0.orig/server/configure	2006-10-25 01:39:43.000000000 +0200
+++ gkrellm-2.3.0/server/configure	2007-10-26 02:02:20.000000000 +0200
@@ -29,19 +29,10 @@
 echo "Checking for libsensors... " 1>& 5
 
 cat << EOF > test.c
-#include <stdio.h>
 #include <sensors/sensors.h>
 
 int main()
 	{
-	FILE	*f;
-
-	f = fopen("/etc/sensors.conf", "r");
-	if (!f)
-		return 1;
-	if (sensors_init(f) != 0)
-		return 1;
-	fclose(f);
 	return 0;
 	}
 EOF
--- gkrellm-2.3.2/src/configure.orig	2008-10-03 23:44:08.000000000 +0100
+++ gkrellm-2.3.2/src/configure	2009-01-19 18:57:39.000000000 +0000
@@ -158,26 +158,11 @@
 echo "Checking for libsensors... " 1>& 5
 
 cat << EOF > test.c
-#include <stdio.h>
 #include <sensors/sensors.h>
 
 int main()
 	{
-#if SENSORS_API_VERSION < 0x400 /* libsensors 3 code */
-	FILE	*f;
-
-	f = fopen("/etc/sensors.conf", "r");
-	if (!f)
-		return 1;
-	if (sensors_init(f) != 0)
-		return 1;
-	fclose(f);
-	return 0;
-#else /* libsensors 4 code */
-	if (sensors_init(NULL) != 0)
-		return 1;
 	return 0;
-#endif
 	}
 EOF
 
