diff -u -r gkrellm-2.3.0.orig/server/Makefile gkrellm-2.3.0/server/Makefile
--- gkrellm-2.3.0.orig/server/Makefile	2007-05-28 21:12:20.000000000 +0200
+++ gkrellm-2.3.0/server/Makefile	2007-10-26 03:45:08.000000000 +0200
@@ -21,7 +21,7 @@
 MANMODE ?= 644
 MANDIRMODE ?= 755
 INSTALL ?= install
-LINK_FLAGS ?= -Wl,-E
+override LINK_FLAGS += -Wl,-E
 EXTRAOBJS =
 
 
@@ -112,7 +112,7 @@
 all:	gkrellmd
 
 gkrellmd: $(OBJS) $(EXTRAOBJS)
-	$(CC) $(OBJS) $(EXTRAOBJS) -o gkrellmd $(LIBS) $(LINK_FLAGS)
+	$(CC) $(LINK_FLAGS) $(OBJS) $(EXTRAOBJS) -o gkrellmd $(LIBS)
 
 static: $(OBJS) $(EXTRAOBJS)
 	$(CC) $(OBJS) $(EXTRAOBJS) -o gkrellmd.static -static \
Only in gkrellm-2.3.0/server: Makefile~
Only in gkrellm-2.3.0/server: .Makefile.swp
diff -u -r gkrellm-2.3.0.orig/src/Makefile gkrellm-2.3.0/src/Makefile
--- gkrellm-2.3.0.orig/src/Makefile	2007-05-28 21:12:20.000000000 +0200
+++ gkrellm-2.3.0/src/Makefile	2007-10-26 03:39:59.000000000 +0200
@@ -18,7 +18,7 @@
 MANMODE ?= 644
 MANDIRMODE ?= 755
 INSTALL ?= install
-LINK_FLAGS ?= -Wl,-E
+override LINK_FLAGS += -Wl,-E
 SMC_LIBS ?= -L/usr/X11R6/lib -lSM -lICE
 
 ifeq ($(without-gnutls),1)
@@ -111,7 +112,7 @@
 all:	gkrellm
 
 gkrellm: $(OBJS) $(UNIXOBJS) $(EXTRAOBJS)
-	$(CC) $(OBJS) $(UNIXOBJS) $(EXTRAOBJS) -o gkrellm $(LIBS) $(LINK_FLAGS)
+	$(CC) $(LINK_FLAGS) $(OBJS) $(UNIXOBJS) $(EXTRAOBJS) -o gkrellm $(LIBS) 
 
 static: $(OBJS) $(UNIXOBJS) $(EXTRAOBJS)
 	$(CC) $(OBJS) $(UNIXOBJS) $(EXTRAOBJS) -o gkrellm.static -static \
Only in gkrellm-2.3.0/src: Makefile.orig
Only in gkrellm-2.3.0/src: .Makefile.swp
