--- gkrellm-2.3.5/src/Makefile~	2010-10-02 18:21:01.000000000 +0200
+++ gkrellm-2.3.5/src/Makefile	2010-10-14 22:28:29.188002756 +0200
@@ -72,7 +72,7 @@
 PKG_LIB = `$(PKG_CONFIG) --libs gtk+-2.0 gthread-2.0`
 
 FLAGS = -O2 -I.. -I$(SHARED_PATH) $(PKG_INCLUDE) $(GTOP_INCLUDE) $(PTHREAD_INC)\
- ${SSL_INCLUDE} ${NTLM_INCLUDE} -DGKRELLM_CLIENT
+ ${SSL_INCLUDE} ${NTLM_INCLUDE} -DGKRELLM_CLIENT ${MYFLAGS}
 
 LIBS = $(PKG_LIB) $(GTOP_LIBS) $(X11_LIBS) $(SYS_LIBS) $(SSL_LIBS) $(SENSORS_LIBS)\
  $(NTLM_LIBS) $(MATH_LIB)
--- gkrellm-2.3.5/server/Makefile~	2010-10-02 18:21:01.000000000 +0200
+++ gkrellm-2.3.5/server/Makefile	2010-10-14 22:32:33.201146284 +0200
@@ -60,7 +60,7 @@
 GLIB12_INCLUDE = `glib-config --cflags`
 GLIB12_LIB = `glib-config --libs glib gmodule`
 
-FLAGS = -O2 $(PKG_INCLUDE)
+FLAGS = -O2 $(PKG_INCLUDE) ${MYFLAGS}
 
 ifeq ($(glib12),1)
     FLAGS = -O2 $(GLIB12_INCLUDE)
